src/data/store/bulk.js

```diff
@@ -19,9 +20,11 @@ export function bulkCreate(objects, asyncHandler) {
     }
   })
 
+  const preparedObjects = objects.map(obj => convertObject(obj))
```

Comment:
I see that `convertObject` returns String, so here we will have an array of JSON strings which will be stringified one more time before sending to the server   

```
JSON.stringify([JSON.stringify({foo:'bar'})]) // => "["{\"foo\":\"bar\"}"]"
```

is this by design?

Reply:
Removed redundant stringify